src/Elasticsearch.Net/Transport/Pipeline/ResponseBuilder.cs

```diff
@@ -27,6 +27,7 @@ public static class ResponseBuilder
 		{
 			responseStream.ThrowIfNull(nameof(responseStream));
 			var details = Initialize(requestData, ex, statusCode, warnings, mimeType);
+			//TODO take ex and (responseStream == Stream.Null) into account might not need to flow to SetBody in that case
```

Comment:
TODO?

Reply:
TODO's left in there to address at a later time :smile_cat: